androidsdk/src/main/java/com/wootric/androidsdk/views/phone/RatingBar.java

```diff
@@ -94,15 +100,20 @@ private void init(Context context) {
     private void initResources() {
         Resources res = mContext.getResources();
 
+        mScaleMinimum = mSurveyType == null ? 0 : Score.minimumScore(mSurveyType);
```

Comment:
Maybe you can define a default value inside the hash for whenever mSurveyType is `null` so it returns 0-10 from there